src/Game.js

```diff
@@ -212,6 +225,32 @@ class Game extends Component {
                 &nbsp;
                 <strong data-testid="moveCounter">{this.moves}</strong>
               </p>
+              <div className="gamepad">
+                <button 
+                className="gamepad__control gamepad__control--up"
+                onClick={this.clickHandler}
+                data-key="ArrowUp">
```

Comment:
The `keyHandler` and `clickHandler` functions do largely the same thing. For this reason duplicating the two doesn't look like a good idea.

Also using data attributes for getting data is good but we can just inline functions for that. It looks more idiomatic in Reactland.

So here's what you should do-

1. Remove `clickHandler` function.
2. Rename `keyHandler` function  to `keyAndClickHandler`.
3. For clicks in gamepage control buttons do this

```js
<button
  className="gamepad__control gamepad__control--up"
  onClick={() => {
    const event = { key: 'ArrowUp' };
    keyAndClickHandler(event);
  }}
</button>
```